schedule/yast/yast2_cmd.yaml

```diff
@@ -2,7 +2,22 @@ name:           yast2_cmd
 description:    >
     Yast2 cmd interface tests.
 
+conditional_schedule:
+  bootloader_zkvm:
+    ARCH:
+      s390x:
+        - installation/bootloader_zkvm
 schedule:
+     # Called on ARCH: s390x
+     - {{bootloader_zkvm}}
```

Comment:
Can we reuse `bootloader_start`? Which will do nothing in all other cases. I guess we need some wrapper that works for booting images. (Out of scope for this ticket though).